### PR TITLE
Merging staging to production

### PR DESCRIPTION
**Pull request by edisinovcic**
Due to the changes in how docker compose is now partitioned as one service one yaml file I will have to update it a bit now manually after this is merged.

### CODE REVIEW

**Comment by edisinovcic** [comment]
Tested manually the flow, everything works. Ready to be merger.

File: src/injector.ts [hunk 1]
```diff
@@ -190,58 +352,86 @@ export default class Injector {
    * @param  {string}            chain      chain name (ex: 'ropsten')
    * @param  {string}            address    contract address
    * @param  {string[]}          files
-   * @return {Promise<string[]>}            addresses of successfully verified contracts
+   * @return {Promise<void>}
```
Comment (cgewecke) [inline src/injector.ts:355]: Forgot to update docs comment here...
```suggestion
   * @return {Promise<string[]>}            addresses of successfully verified contracts
```

File: environments/.env.stable [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+BUCKET_NAME_STAGING=s3://staging-source-verify
+
+# Geth config
+NETWORK_NAME=goerli
+
+# IPFS config
+IPFS_SECRET=xxx
+
+# Other config
+TESTING=true
```
Comment (cgewecke) [inline environments/.env.stable:41]: Maybe...
```suggestion
TESTING=false
```

File: environments/.env.latest [hunk 3]
```diff
@@ -1,22 +1,43 @@
...
 
+# Geth config
+NETWORK_NAME=goerli
+
+# IPFS config
+IPFS_SECRET=xxx
 
 # Other config
-#can be stable for production or latest for development
 TESTING=true
```
Comment (cgewecke) [inline environments/.env.latest:41]: Setting TESTING to true make the monitor listen to localchain **only**. 
```suggestion
TESTING=false
```

**Pull request merged**